src/Symfony/Bundle/FrameworkBundle/Resources/config/console.php

```diff
@@ -0,0 +1,290 @@
+            ->tag('console.command', ['command' => 'debug:messenger'])
+
+        ->set('console.command.messenger_stop_workers', StopWorkersCommand::class)
+            ->args([
+                service('cache.messenger.restart_workers_signal'),
+            ])
+            ->tag('console.command', ['command' => 'messenger:stop-workers'])
+
+        ->set('console.command.messenger_failed_messages_retry', FailedMessagesRetryCommand::class)
+            ->args([
+                abstract_arg('Receiver name'),
+                abstract_arg('Receiver'),
+                service('messenger.routable_message_bus'),
+                service('event_dispatcher'),
+                service('logger'),
+            ])
+            ->tag('console.command', ['command' => 'messenger:failed:retry'])
+
+        ->set('console.command.messenger_failed_messages_show', FailedMessagesShowCommand::class)
+            ->args([
+                abstract_arg('Receiver name'),
+                abstract_arg('Receiver'),
+            ])
+            ->tag('console.command', ['command' => 'messenger:failed:show'])
+
+        ->set('console.command.messenger_failed_messages_remove', FailedMessagesRemoveCommand::class)
+            ->args([
+                abstract_arg('Receiver name'),
+                abstract_arg('Receiver'),
+            ])
+            ->tag('console.command', ['command' => 'messenger:failed:remove'])
+
+        ->set('console.command.router_debug', RouterDebugCommand::class)
+            ->args([
+                service('router'),
+                service('debug.file_link_formatter')->nullOnInvalid(),
+            ])
+            ->tag('console.command', ['command' => 'debug:router'])
+
+        ->set('console.command.router_match', RouterMatchCommand::class)
+            ->args([
+                service('router'),
+                tagged_iterator('routing.expression_language_provider'),
+            ])
+            ->tag('console.command', ['command' => 'router:match'])
+
+        ->set('console.command.translation_debug', TranslationDebugCommand::class)
+            ->args([
+                service('translator'),
+                service('translation.reader'),
+                service('translation.extractor'),
+                param('translator.default_path'),
+                abstract_arg('twig.default_path'),
+                abstract_arg('Translator paths'),
+                [],
+            ])
+            ->tag('console.command', ['command' => 'debug:translation'])
+
+        ->set('console.command.translation_update', TranslationUpdateCommand::class)
+            ->args([
+                service('translation.writer'),
+                service('translation.reader'),
+                service('translation.extractor'),
+                param('kernel.default_locale'),
+                param('translator.default_path'),
+                abstract_arg('twig.default_path'),
+                abstract_arg('Translator paths'),
+                [],
```

Comment:
non-existent argument ?

Reply:
oh no, missing `abstract_arg('Twig paths')`

Reply:
Yes, updating this back but somehow this break the testing is this okay?!

Reply:
Yes, sorry for that fixed :)